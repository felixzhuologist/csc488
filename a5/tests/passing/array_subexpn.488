{
    var K[8] : Boolean
    var B[12] : Integer
    var A[-4 .. 10] : Integer

    K[1] := true or false
    B[1] := -21
    B[2] := -22
    B[3] := -23
    B[12] := -212
    A[-4] := B[3] + 10
    A[-3] := 13
    A[10] := 12
    
    write A[-4], newline
    write A[-3], newline
    write A[10], newline
    write B[1], newline
    write B[2], newline
    write B[3], newline
    write B[12], newline
}