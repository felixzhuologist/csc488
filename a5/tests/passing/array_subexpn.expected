-13
13
12
-21
-22
-23
-212
